packages/eslint-config-expo/utils/core.js

```diff
@@ -46,8 +46,9 @@ module.exports = {
     'unicode-bom': ['warn', 'never'],
     'use-isnan': 'error',
     'valid-typeof': 'error',
-    'import/first': 'warn', //keep
+    'import/first': 'warn',
     'import/default': 'off',
+    'import/no-unresolved': 'off',
```

Comment:
Thinking about this some more, I think the template project should deal with this (the imports) in its own linter config by telling ESLint about the `@` convention. Since the template is diverging from standard behavior, it is also the template's responsibility to set up the linter rules to accommodate that behavior.

Reply:
That would work, though our templates don't have eslint set up currently. Should we add eslint to the tabs template?

Reply:
We could do a few things:

1. Fully add ESLint to the tabs template out of the box.
2. Add an inactive ESLint config entry to the template's package.json or give the template an ESLint config file that would become active if the developer chose to add ESLint to their project.
3. Change the tabs template to use standard import specifiers.

I didn't set up the templates and don't have the clearest opinions around them. The thing that is most important to me is that the template should be responsible for configuring ESLint to work with non-standard import specifiers.

Perhaps we could look at how tsconfig.json is set up for the template project -- tsc also needs to be told how to handle non-standard import specifiers.